cluster/defaults.go

```diff
@@ -71,6 +71,17 @@ const (
 
 	DefaultCanalFlexVolPluginDirectory = "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/nodeagent~uds"
 
+	DefaultApicRefreshTime                           = 1200
+	DefaultOVSMemoryLimit                            = "1Gi"
+	DefaultImagePullPolicy                           = "Always"
+	DefaultServiceMonitorInterval                    = 0
+	DefaultPBRTrackingNonSnat                        = "false"
+	DefaultInstallIstio                              = "false"
+	DefaultIstioProfile                              = "demo"
+	DefaultDropLogEnable                             = "true"
```

Comment:
Is DropLog and debug going to be spammy for a default setup? If these are the ACI defaults then it's fine, otherwise please set the default setting/level.

Reply:
Thanks, dropLog enabled by default is an ACI-CNI default setting, other debugs should be info level, fixed these